src/js/utils/language.js

```diff
@@ -36,8 +36,12 @@ function normalizeLanguageCode(language) {
     return underscoreIndex === -1 ? languageAndCountryCode : languageAndCountryCode.substring(0, underscoreIndex);
 }
 
+function normalizeNorwegian(language) {
+    return /^n(?:b|n)_no$/.test(language) ? `no_${language.substring(0, 2)}` : language;
```

Comment:
```suggestion
    return language.replace(/^n(b|n)_no$/, 'no_n$1')
```

Reply:
If we return country_language here, for nb-NO and nn-NO page language settings it might result in us loading the correct default, but it will break custom config `intl.nb_no` and `intl.nn_no` blocks.

Rather than flipping language and code, just normalize/map "nn" and "nb" to no at the point where we pick an intl block as a fallback.

You just need to add them to the list used by `isTranslationAvailable` and then map the Norwegian languages to "no" for the file, not the key we store the language in, in `isLocalizationComplete` and `loadJsonTranslation`.